spacy/lang/la/lex_attrs.py

```diff
@@ -0,0 +1,32 @@
+from ...attrs import LIKE_NUM
+import re
+
+# cf. Goyvaerts/Levithan 2009
+roman_numerals_compile = re.compile(r'^(?=[MDCLXVI])M*D?C{0,4}L?X{0,4}V?I{0,4}$')
```

Comment:
Addressing... fwiw in Latin texts I would expect uppercase (or small caps) more often than not, but it does seem worthwhile to include lowercase, esp. since there is likely a wide variation in the preprocessing of digitized texts (see, e.g., https://www.thelatinlibrary.com/histbrit.html)